specification/2.0/appendix/performance-metrics/README.md

```diff
@@ -0,0 +1,31 @@
+
```

Comment:
Minor Comment: May be we should have a description saying: The goal of these metrics is to assess "Runtime" performance  of the asset.

Reply:
Good point - I will add.
I also had some feedback from Don that I will add.